diff -Nur alsa-patch-bay-0.5.1/src/Makefile.am alsa-patch-bay-0.5.1-makefile/src/Makefile.am
--- alsa-patch-bay-0.5.1/src/Makefile.am	Fri Feb  7 22:48:00 2003
+++ alsa-patch-bay-0.5.1-makefile/src/Makefile.am	Mon Apr 28 21:36:40 2003
@@ -35,4 +35,4 @@
 EXTRA_DIST = 5-pin-din.png 5-pin-din.xbm
 
 install-exec-local:
-	$(LN_S) -f $(bindir)/alsa-patch-bay $(bindir)/jack-patch-bay
+	$(LN_S) -f $(bindir)/alsa-patch-bay $(DESTDIR)$(bindir)/jack-patch-bay
